modules/http4s/src/smithy4s/http4s/SimpleRestJsonBuilder.scala

```diff
@@ -17,47 +17,67 @@
 package smithy4s
 package http4s
 
-object SimpleRestJsonBuilder extends SimpleRestJsonBuilder(1024, false, true)
+import smithy4s.json.Json
+import smithy4s.json.JsonPayloadCodecCompiler
+
+object SimpleRestJsonBuilder
+    extends SimpleRestJsonBuilder(
+      new internals.SimpleRestJsonCodecs(
+        jsonCodecs = Json.payloadCodecs,
+        explicitDefaultsEncoding = false,
+        hostPrefixInjection = true
+      )
+    )
 
 class SimpleRestJsonBuilder private (
     simpleRestJsonCodecs: internals.SimpleRestJsonCodecs
 ) extends SimpleProtocolBuilder[alloy.SimpleRestJson](
       simpleRestJsonCodecs
     ) {
 
+  @deprecated(message = "Use builder methods instead", since = "0.18.25")
```

Comment:
**suggestion:** as we don't have an "actual" builder, maybe

```suggestion
  @deprecated(message = "Use .withXXX methods instead", since = "0.18.25")
```